test/e2e/framework/auth/helpers.go

```diff
@@ -43,6 +43,29 @@ type bindingsGetter interface {
 	v1rbac.ClusterRolesGetter
 }
 
+// WaitForSubresourceAuthorizationUpdate checks if the give user can preform the
+// named verb and action on the subresource.
+func WaitForSubresourceAuthorizationUpdate(ctx context.Context, c v1authorization.SubjectAccessReviewsGetter, user string, ra *authorizationv1.ResourceAttributes, allowed bool) error {
```

Comment:
nit: this isn't really specific to Subresources anymore. Maybe `WaitForAuthzUpdate` or just `WaitForAuthorization`?

Reply:
updated to WaitForAuthzUpdate, thanks!